README.md

```diff
@@ -48,3 +48,5 @@
 * Places called "Fish Tacos"
 * Going south of Alameda
 * Stone Temple Pilots
+* DC Stickers on cars
+* People turning on the office lights
```

Comment:
I'm not sure DC stickers rise to Kishtonite, but I am def not a fan.
"People turning on the office lights": this is redundant to "Low-lit work areas"